tests/test-math.c

```diff
@@ -111,6 +111,118 @@ int main(int argc, char** argv)
       }",
       "CCABACC");
 
+  assert_true_rule_blob(
```

Comment:
In cases where the rule doesn't uses the data for anything, you can use `assert_true_rule` instead of `assert_true_rule_blob`

Reply:
Modified as recommended. Please review it again.